src/mri/exp_mult_mex.jl

```diff
@@ -0,0 +1,162 @@
+using Random
+#using MIRT: max_percent_diff
+#using MIRT: exp_xform
+
+export exp_mult_mex
```

Comment:
"mex" is a matlab term.  we don't want any matlab terms in this code.
please remove `_mex` everywhere, including in the filename!